soh/include/functions.h

```diff
@@ -2418,7 +2418,10 @@ OSThread* __osGetCurrFaultedThread(void);
 u32* osViGetCurrentFramebuffer(void);
 s32 __osSpSetPc(void* pc);
 f32 absf(f32);
-void* func_801068B0(void* dst, void* src, size_t size);
+#ifndef __cplusplus
+void* oot_memmove(void* dest, const void* src, size_t len);
+#define memmove oot_memmove
+#endif
```

Comment:
Instead of doing this, can we just move the define to `__osMalloc.c`, that way all non-decomp calls to `memmove` use the real one?

Reply:
Fixed.